packages/api/src/parse-cli.ts

```diff
@@ -0,0 +1,220 @@
+import yargs from "yargs";
+import { CamelKeys } from "./types/common";
+
+function coerceArr(arg) {
```

Comment:
Ugh just noticed Github didn't catch the rename here. I believe it's because the `parsed` intermediate variable in the `parceCli` function (used for typing purposes) caused a too-big indentation change.

Reviewer, let me know if you want me to put this change on a separate commit or something so it's more reviewable.

Reply:
OK, you can check the real changes made to this file, together with the related changes on this commit: 

https://github.com/livepeer/livepeer-com/pull/476/commits/b4e80906a132a7bc42016ba222d7470016a8cc4b?w=1

Link already ignoring whitespace changes which are basically prettier itself 👌